app/controllers/course-admin/tester-version.ts

```diff
@@ -0,0 +1,82 @@
+import Controller from '@ember/controller';
+import CourseTesterVersionModel from 'codecrafters-frontend/models/course-tester-version';
+import Store from '@ember-data/store';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class CourseAdminTesterVersionController extends Controller {
+  declare model: {
```

Comment:
Since model is used by both `Controller` and `Route`, one pattern I've been following is to defined `ModelType` in the route, and then import it in the controller. That way things don't accidentally go out of sync. 

Example: https://github.com/codecrafters-io/frontend/blob/6efa728927156a6420ef454803a1a9935e8fc9ed/app/routes/catalog.ts#L10